### ISSUE TITLE
"Could not load necessary function to initialize OpenGL context" after 21af6fec

### ISSUE THREAD

**Issue by xparq** [SFML/SFML#2497]
## "Could not load necessary function to initialize OpenGL context" error on start, after 21af6fec

SFML-3 apps that used to work fine now fail to start after the change in the Windows OpenGL init code.

## Your environment

* Windows 10
* SFML 485d367, static libs
* VS 2022
* 
  ~~~~
  cl -MD -O2 -DNDEBUG -nologo -c -W4 -Iinclude -DSFML_STATIC -EHsc -std:c++latest demo.cpp
  link -out:demo.exe sfml-graphics-s.lib sfml-window-s.lib sfml-system-s.lib opengl32.lib freetype.lib user32.lib kernel32.lib gdi32.lib winmm.lib advapi32.lib demo.obj
  ~~~~

E.g.:

    #include <SFML/Graphics.hpp>
    int main()
    {
        sf::RenderWindow window(sf::VideoMode({800, 600}), "");
        return 0;
    }

Console output:

    Could not load necessary function to initialize OpenGL context
    Could not load necessary function to initialize OpenGL context
    Setting vertical sync not supported

_(Sorry if I'm missing something important, I'm late with other duties already, but wanted to still give you guys a quick heads-up, erring on the side of assuming this might be helpful as-is, nonetheless.)_

**Comment by ChrisThrasher** [comment]
Is this a duplicate of #2495?

**Comment by xparq** [comment]
(Updated the comment above -- forgot to copy the console output.)

**Comment by ChrisThrasher** [comment]
Closed as duplicate of #2495
